src/components/menu/menu.ts

```diff
@@ -205,6 +208,8 @@ export class Menu implements RootNode {
   private _events: UIEventManager;
   private _gestureBlocker: BlockerDelegate;
   private _isPane: boolean = false;
+  private _side: side = 'start';
```

Comment:
The idea behind having `_side` and `@Input() side` is that the `_side` has 4 possible values: `['left', 'right', 'start', 'end']`
While the private magic variable `side` has only two values: `['left', 'right']`
This makes it easier for the rest of that file. Every time there is a reference to `side`, it will evaluate `_side` and return the correct one for that specific call. `start` can return `left` or `right`, depending on platform, etc..